kolla-ansible/ansible/roles/triliovault/templates/triliovault-datamover.conf.j2

```diff
@@ -61,4 +67,7 @@ helper_command = sudo /usr/bin/privsep-helper
 {% endif %}
 
 [conductor]
-use_local = True
\ No newline at end of file
+use_local = True
+
+[oslo_messaging_rabbit]
+ssl = False
```

Comment:
@trilio-shiwanand : Do not hardcode ssl=False. Check if kolla ansible has variable for it.